cachito/workers/pkg_managers/pip.py

```diff
@@ -1965,7 +1966,7 @@ def _push_downloaded_requirement(requirement, pip_repo_name, raw_repo_name):
     if requirement["kind"] == "pypi":
         try:
             upload_pypi_package(pip_repo_name, requirement["path"])
-        except (NetworkError, ValueError):
+        except (UploadError, NetworkError, ValueError):
```

Comment:
Checking if we can remove NetworkError from these places

Reply:
I think so, +1 to remove it. The ruby code catches only `UploadError`